### PR TITLE
Change moment to date-fns

### PR DESCRIPTION
**Pull request by pusztaienike**
image

### CODE REVIEW

File: apps/sensenet/src/hooks/use-date-utils.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  const personalSettings = usePersonalSettings()
+
+  const formatDistanceFromNow = (date: any) => {
+    const isHungarian = personalSettings.language === 'hungarian'
+    return formatDistanceToNow(date, { locale: isHungarian ? hu : enUS, addSuffix: true })
+  }
+
+  return {
+    /**
+     * Parse date
```
Comment (taki9) [inline apps/sensenet/src/hooks/use-date-utils.ts:31]: you can remove these comments here, don't give additional info

File: apps/sensenet/src/components/view-controls/browse-view.tsx [hunk 2]
```diff
@@ -55,6 +58,7 @@ export const BrowseView: React.FC<BrowseViewProps> = (props) => {
         handleCancel={() => navigateToAction({ history, routeMatch })}
         controlMapper={controlMapper}
         localization={{ cancel: localization.forms.cancel }}
+        locale={personalSettings.language === 'hungarian' ? hu : enUS}
```
Comment (taki9) [inline apps/sensenet/src/components/view-controls/browse-view.tsx:61]: we should create a function on LocalizationContext, which returns the locale (this approach is not sustainable if we have more languages available).

Comment (pusztaienike) [reply apps/sensenet/src/components/view-controls/browse-view.tsx:61]: I used an object there to define the languages, the langcodes and the locale. I hope it is also good :)

File: apps/sensenet/src/hooks/use-date-utils.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import format from 'date-fns/format'
+import formatDistanceToNow from 'date-fns/formatDistanceToNow'
+import enUS from 'date-fns/locale/en-US'
+import hu from 'date-fns/locale/hu'
+import parseISO from 'date-fns/parseISO'
+import { usePersonalSettings } from '.'
+
+export const parseDate = (date: any) => {
```
[the remark this answers is not in the document]
Comment (pusztaienike) [reply apps/sensenet/src/hooks/use-date-utils.ts:8]: It should be number | Date than, cause the date-fns functions expect the parameter with that types:
image

**Review by taki9 — APPROVED** [review]

**Review by herflis — APPROVED** [review]

**Pull request merged**